src/components/Profile/Profile.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import s from './Profile.module.css';
```

Comment:
Consider renaming "s" to "styles" to be more descriptive.

Reply:
/fxd